src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -90,7 +90,6 @@
         }
         @await RenderBodyAsync()
     </div>
-    </div>
```

Comment:
Why is this change in this pr? Build is currently broken

Reply:
I have no idea! Unintentional error